docs/rm-eq.rst

```diff
@@ -1,9 +1,64 @@
-Securely creating survey respondent sessions
---------------------------------------------
+Respondent survey session schema
+--------------------------------
 
+When a respondent is ready to take a survey hosted on the eq system, a set of details
+need to be passed to eq to setup the survey correctly. This data is wrapped inside a json web
+token which is attached to the end of a url, digitally signed and authorised by a respondent
+management system. This creates a clean interface for any respondent management system
+to integrate with the eq system.
 
-JWT
+Schema definition
+  user_id
+    The id assigned by the respondent management system
+  ru_ref
+    The responding unit reference id.
+  eq_id
+    The eQ questionnaire instance id.
+  collection_exercise_sid
+    A reference number used to represent the collection exercise inside the ONS
+  period_id
+    A numerical reference to either a month or quarter time period
+  period_str
+    A display name for the period referenced above
+  ref_p_start_date
+    A start date for the relevant reference period
+  ref_p_end_date
+    An end date for the relevant reference period
+  form_type
+    The particular form_type for a responding unit
```

Comment:
Are these all Mandatory? Should we be rejecting the JWT if it doesn't include them?